flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/operations/ExecutableOperation.java

```diff
@@ -57,5 +59,9 @@ interface Context {
         FunctionCatalog getFunctionCatalog();
 
         ModuleManager getModuleManager();
+
+        ResourceManager getResourceManager();
```

Comment:
it looks like we can get catalog and module or jars. how about showJobs ?

Reply:
Currently, `ShowJobsOperation` is implemented in SQL Gateway, not in `TableEnvironemntImpl`. And it relies on the additional `JobClient` to fetch jobs that require more effort. So I would like to migrate it in another issue.